server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/Mailbox.java

```diff
@@ -341,7 +356,8 @@ public final boolean equals(Object obj) {
                 && Objects.equals(this.totalThreads, other.totalThreads)
                 && Objects.equals(this.unreadThreads, other.unreadThreads)
                 && Objects.equals(this.sharedWith, other.sharedWith)
-                && Objects.equals(this.namespace, other.namespace);
+                && Objects.equals(this.namespace, other.namespace)
```

Comment:
`namespace` wasn't here? We don't have EqualsVerifier on this?

Can we add it?

Reply:
I think you misread, namespace is not added, that's just its line that is changed